features/test/een-ouder-staande-huwelijk.feature

```diff
@@ -0,0 +1,153 @@
+
+  Regel: Als de minderjarige tijdens het huwelijk of partnerschap van de juridische ouder is geadopteerd, dan hebben de ouder en diens (toenmalige) partner gezamenlijk gezag
+
+  Regel: Als de minderjarige tijdens het huwelijk of partnerschap van de juridische ouder geboren is en er sprake is van ontkenning vaderschap, dan heeft de ouder eenhoofdig ouderlijk gezag
+
+  Regel: Als de ouder is overleden of onbevoegd is voor gezag, dan is er voogdij door de partner
+
+  Regel: Als de partner is overleden of onbevoegd is voor gezag, dan is er eenhoofdig ouderlijk gezag
+
+  Regel: Als zowel de ouder als de partner is overleden of onbevoegd is voor gezag, dan is er tijdelijk geen gezag
+
+  Regel: De partner behoudt gezag ook na nietigverklaring van het huwelijk
+    De nietigverklaring heeft terugwerkende kracht tot het tijdstip van de huwelijkssluiting of het aangaan van het geregistreerde partnerschap.
+    Het huwelijk of het geregistreerde partnerschap wordt geacht niet te hebben bestaan.
+    Een rechtsfeit dat heeft plaatsgevonden voor de nietigverklaring, behoudt het rechtsgevolg.
+
+    Een nietigverklaring van een huwelijk is ongelijk aan een nietig huwelijk.
+    Een nietig huwelijk of geregistreerd partnerschap heeft geen enkel rechtsgevolg.
+
+    | vraag | logische situatie                                         | verwacht route | verwacht resultaat         |
+    | 2b.1  | huwelijk is na geboorte nietig verklaard                  | 41             | gezamenlijk gezag          |
+    | 2b.1  | huwelijk is voor geboorte nietig verklaard                |                | eenhoofdig ouderlijk gezag |
+    | 2b.1  | huwelijk is na geboorte nietig (verwijderd met correctie) |                | eenhoofdig ouderlijk gezag |
+    
+    Deze regel is nog niet geïmplementeerd!
+
+    @nieuw
+    Scenario: huwelijk is na geboorte nietig verklaard
+      Gegeven de persoon 'partner' met burgerservicenummer '000000024'
+      * is meerderjarig
+      En de minderjarige persoon 'minderjarige' met één ouder 'moeder'
+      * is vandaag - 8 jaar geboren
+      En 'moeder' en 'partner' zijn morgen - 9 jaar gehuwd
+      En het huwelijk van 'moeder' en 'partner' is morgen - 8 jaar nietig verklaard
+      Als 'gezag' wordt gevraagd van 'minderjarige'
+      Dan is het gezag over 'minderjarige' gezamenlijk gezag met ouder 'moeder' en derde 'partner'
+
+    @deprecated
+    Scenario: huwelijk is na geboorte nietig verklaard
+      Gegeven de persoon 'partner' met burgerservicenummer '000000024'
+      * is meerderjarig
+      En de minderjarige persoon 'minderjarige' met één ouder 'moeder'
+      * is vandaag - 8 jaar geboren
+      En 'moeder' en 'partner' zijn morgen - 9 jaar gehuwd
+      En het huwelijk van 'moeder' en 'partner' is 8 jaar geleden nietig verklaard
+      Als 'gezag' wordt gevraagd van 'minderjarige'
+      Dan is het gezag over 'minderjarige' eenhoofdig ouderlijk gezag met ouder 'moeder'
```

Comment:
Als ik het goed begrijp, wordt hier geprobeerd het volgende te zeggen: "deze regel werkt momenteel niet zoals het hoort en is nog niet geïmplementeerd (deprecated), maar hoort in de toekomst zo te werken (nieuw)". 

1) Naar mijn gevoel gaat dat buiten een regressie test suite om
2) `@deprecated` zorgt ervoor dat de test wordt gerund met 2.6 parameters in plaats van 2.7 parameters. `@nieuw` zorgt ervoor dat de test wordt gerund met de "huidige" (2.7) parameters. Volgens mij is dit geen kwestie van 2.6 vs 2.7, aangezien nietig verklaring van huwelijken niet in 2.7 zit.

Reply:
@nieuw moet hier zijn @todo @skip-verify